main.cpp

```diff
@@ -47,58 +52,55 @@ int main(int argc, char *argv[]) {
 
     int mode = atoi(argv[2]);
     std::string JASMINEGRAPH_HOME = Utils::getJasmineGraphHome();
-    std::string profile = argv[1];  // This can be either "docker" or "native"
+    jasminegraph_profile = strcmp(argv[1], "docker") == 0 ? PROFILE_DOCKER : PROFILE_K8S;
     std::string enableNmon = "false";
 
     main_logger.log("Using JASMINE_GRAPH_HOME", "info");
```

Comment:
Even though this is not part of this PR we should fix this log line and we should remove printing the `JASMINE_GRAPH_HOME` variable into console. Instead it should be part of the logline.

Reply:
Fixed

Reply:
Ok